mmtrack/datasets/mot_challenge_dataset.py

```diff
@@ -240,6 +245,73 @@ def format_bbox_results(self, results, infos, resfile):
                         f'{(y2-y1):.3f},{conf:.3f}\n')
             f.close()
 
+    def get_dataset_cfg_for_hota(self, tracker_folder, seqmap):
+        """Get default configs for trackeval.datasets.MotChallenge2DBox.
+
+        Args:
+            tracker_folder (str): the name of the tracker folder
+
+        Returns:
+            Dataset Configs for MotChallenge2DBox.
+        """
+        dataset_config = dict(
+            # Location of GT data
+            GT_FOLDER=self.img_prefix,
+            # Trackers location
+            TRACKERS_FOLDER=tracker_folder,
+            # Where to save eval results
+            # (if None, same as TRACKERS_FOLDER)
+            OUTPUT_FOLDER=None,
+            # Use 'track' as the default tracker
+            TRACKERS_TO_EVAL=['track'],
+            # Option values: ['pedestrian']
+            CLASSES_TO_EVAL=['pedestrian'],
+            # Option Values: 'MOT17', 'MOT16', 'MOT20', 'MOT15'
+            BENCHMARK=self.img_prefix.rsplit('/', 2)[-2],
```

Comment:
This is risky, write a function to convert lower-case names, and raise errors if there are no specific benchmarks in their names.